### PR TITLE
Created modals

### PR DESCRIPTION
**Pull request by alphayowakarindi**
In this pull request, we created the modals to display more details of a portfolio on desktop and mobile. After that, we used Javascript to implement the logic to show and hide the modals when a button to see more details is clicked and to also display the portfolio section dynamically on mobile and desktop.

### CODE REVIEW

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @alphayowakarindi,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- GitHub flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters is set correctly  ✔️ 
-  When the user clicks the button to check project details, the popup with details about the project appears.✔️   
- When the user clicks the close button, the popup disappears ✔️ 
- You have followed  JavaScript best practices. ✔️ 
- The projects section is created dynamically with the information from the projects array.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Kindly, it would be nice to add explicit steps for cloning and running your app, as this will help you, non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`, etc.

- [ ] I would also recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```
- Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.js [hunk 1]
```diff
@@ -1,29 +1,158 @@
...
+    id: 1,
+    name: "Tonic",
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required.",
+    featured_img: "./images/desktop-works/work-one.png",
+    tecnologies: ["html", "css", "javascript"],
+    live_link: "https://alphayowakarindi.github.io/My-Portfolio-1/",
+    source_link: "https://github.com/alphayowakarindi/My-Portfolio-1",
+  },
+  {
```
[the remark this answers is not in the document]
Comment (alphayowakarindi) [reply main.js:43]: @BenMukebo Thank you very much for the recommendations to make my code better. Cheers!

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi Team

Good job working on the changes asked by the previous reviewer. 👍👍👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights
✅ Correct use of Github workflow
✅ No linter error
✅  README is okay
✅ Modal works fine.

## Optional Changes
- [ ] Kindly adjust the broken image for closing the modal.
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by alphayowakarindi** [comment]
> # Status: Approved 
> Hi Team
> 
> Good job working on the changes asked by the previous reviewer. +1+1+1
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it: ship it: Congratulations! tada
> 
> ## Highlights
> white_check_mark Correct use of Github workflow white_check_mark No linter error white_check_mark README is okay white_check_mark Modal works fine.
> 
> ## Optional Changes
> * [ ]  Kindly adjust the broken image for closing the modal.
>   image
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hi @AmaduKamara Thank you very much for the feedback. I've just cloned the project and the image icon is working fine. I think it might be a failure on your side. Kindly try it again and please let me know if the behavior is still the same.

**Comment by alphayowakarindi** [comment]
> It's okay. If it's working fine, no issue then. You can continue with your work. +1

Thank you!

**Pull request merged**